cypress/integration/3-init.spec.js

```diff
@@ -7,7 +7,8 @@
 
 describe('Ladybug simple protractor test', function() {
   it('Confirm title of ladybug app', function() {
-    cy.visit('http://localhost:4200');
-    cy.title().should('match', /Ladybug - v[0-9]+.[0-9]+.[0-9]+/);
+    cy.visit('');
+    // Please update the version here when you update the version in package.json
+    cy.title().should('eq', 'Ladybug - v0.0.15');
```

Comment:
Please revert this to `cy.title().should('match', /Ladybug - v[0-9]+.[0-9]+.[0-9]+/);` so that the test will pass on any version

Reply:
Done.